app/views/DistributionsIncludedView.scala.html

```diff
@@ -0,0 +1,87 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    govukInput : GovukInput,
+    govukDetails : GovukDetails
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@yesHtml = {
+    @govukInput(
+        InputViewModel(
+            field = form("distributionsIncludedAmount"),
+            label = Label(content = Text(messages("distributionsIncluded.amountLabel")))
+        )
+        .asNumeric()
+        .withPrefix(PrefixOrSuffix(content = Text("£")))
+        .withCssClass("govuk-!-width-one-third")
+    )
+}
+
+@layout(pageTitle = title(form, messages("distributionsIncluded.title"))) {
+
+    @formHelper(action = routes.DistributionsIncludedController.onSubmit(mode), 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form,  errorLinkOverrides = Map("value" -> "value_0")))
+        }
+
+        <h1 class="govuk-fieldset__legend--l">
+            @messages("distributionsIncluded.heading")
+        </h1>
+        <p class="govuk-body govuk-!-margin-bottom-4"></p>
+        <p class="govuk-body">@messages("distributionsIncluded.listHeading")</p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>@messages("distributionsIncluded.listItem1")</li>
+            <li>@messages("distributionsIncluded.listItem2")</li>
+            <li>@messages("distributionsIncluded.listItem3")</li>
+        </ul>
+
+        @govukDetails(Details(
+            summary = Text(messages("distributionsIncluded.detailsTitle")),
+            content = HtmlContent(messages("distributionsIncluded.detailsHtml"))
+        ))
+
+        @govukRadios(
+            RadiosViewModel(
+                field  = form("distributionsIncluded"),
+                legend = LegendViewModel(messages("distributionsIncluded.heading")).withCssClass("govuk-visually-hidden"),
+                items = Seq(
+                    RadioItem(
```

Comment:
Remove the label value please. For the label, can we use a generic yes/no e.g site.yes = Yes

Reply:
Done